### PR TITLE
Feature: Colored month in scoreboard

### PR DESCRIPTION
**Pull request by j10a1n15**
Copied from my scoreboard pr #893 
(not related to the custom scoreboard)

It uses the color codes of the atmospheric filter thingy btw

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java [hunk 1]
```diff
@@ -113,6 +113,7 @@ public class MiscConfig {
     @FeatureToggle
     public boolean brewingStandOverlay = true;
 
+    // TODO move into scoreboard accordion
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java:116]: Too early

File: src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java [hunk 2]
```diff
@@ -125,6 +126,12 @@ public class MiscConfig {
     @FeatureToggle
     public boolean hidePiggyScoreboard = true;
 
+    @Expose
+    @ConfigOption(name = "Color Month Names", desc = "Color the month names in the Scoreboard.\nAlso applies to the Custom Scoreboard")
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java]: Dot at the end.

File: src/main/java/at/hannibal2/skyhanni/mixins/hooks/GuiIngameHook.kt [hunk 3]
```diff
@@ -51,5 +51,28 @@ private fun replaceString(text: String): String? {
         }
     }
 
+    if (SkyHanniMod.feature.misc.colorMonthNames) {
+        for (season in Season.values()) {
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/mixins/hooks/GuiIngameHook.kt]: Please use ``Season.entries`` instead

**Review by hannibal002 — CHANGES_REQUESTED** [review]
Code looks clean and works fine. Just small cleanup.

**Comment by j10a1n15** [comment]
Done

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**